package.json

```diff
@@ -29,14 +29,15 @@
         "prop-types": "^15.8.1",
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
-        "react-firebase-hooks": "^4.0.2",
+        "react-firebase-hooks": "^5.0.0",
```

Comment:
generally it's bad that we got the "^" in front of each package version, in real life project the version control should be done manually, in this case it'ssmall project, and packages are pretty much "standard" and "generic" like, so no suprises here I guess even when major version comes out, but at the point when we would get full setup,and we would be sure that all of the packages are compatible witch each other and we can use them without any fear, we shold remove the "^", it's good practice in most cases, you can create a task for that  :P